src/App.tsx

```diff
@@ -1,4 +1,7 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable max-len */
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
should work without these rules
feel free to ask in fe_chat if you need help with it